libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -169,9 +153,7 @@ public void updateDebugLoggingEnabled(boolean isDebugLoggingEnabled) {
   }
 
   public void updateUserAgent(String userAgent) {
-    if (isUserAgentValid(userAgent)) {
```

Comment:
should we keep this check? if `userAgent` is empty or `null` the next line will produce crash.